builds/e2e/e2e.yaml

```diff
@@ -33,6 +33,9 @@ jobs:
       name: $(pool.name)
       demands: rpi3-e2e-tests
 
+    # If Agent name is specified, run only nested edge test
+    condition: eq(variables['agent.name'], '')
```

Comment:
We don't need the conditions in `e2e.yaml` right? That was kind of the point of moving the nested edge job to its own pipeline...